mcp/motors/ec_motors.c

```diff
@@ -1645,15 +1713,6 @@ static int motor_pdo_init(int m_periph)
     if (!ec_SDOwrite32(m_periph, ECAT_RXPDO_MAPPING, 2, map.val)) {
         blast_err("Failed mapping!");
     }
-    // map_pdo(&map, ECAT_LATCHED_DRIVE_FAULT, 32);
-    // if (!ec_SDOwrite32(m_periph, ECAT_RXPDO_MAPPING, 3, map.val)) {
-    //     blast_err("Failed mapping!");
-    // }
-    // Uncomment this once the phasing angle readout has been debugged
```

Comment:
Deleted because bug has been fixed and this is no longer necessary? I.e. why delete this instead of uncomment?

Reply:
This object (0x21C0) is not PDO-mappable. Phase init is decidedly a pre-operation step, so it doesn't make sense to try to map it as a process data object like other thing we'll need to read/write real-time. We set our phase initialization mode for each motor to use encoders only in CME...I'll have to see which mode that corresponds to here for documentation purposes, and maybe to update `rw_init_phasing` or something. This code isn't really valid. Fixed a bug below this though.